src/utils/notifications.ts

```diff
@@ -200,6 +223,32 @@ export const sendExpenditureUpdateNotifications = async ({
   });
 };
 
+// Send a notification when an extension is updated.
+export const sendExtensionUpdateNotifications = async ({
+  colonyAddress,
+  transactionHash,
+  creator,
+  extensionHash,
+  notificationType,
+}: ExtensionUpdateNotificationVariables): Promise<void> => {
+  // Get the recipients of the colony wide notifications.
+  const recipients = await getRecipientsOfColonyWideNotification(colonyAddress);
+
+  if (!recipients.length) {
+    return;
+  }
+
+  // Send the colony wide notifications.
+  await sendNotification(`Extension: ${extensionHash}`, recipients, {
+    notificationType,
+    creator,
+    notificationCategory: NotificationCategory.Extension,
```

Comment:
cc @davecreaser should we use extension as a category or are these product wise included in the `Admin` category?

Reply:
Yeah you're correct, they should all use Admin as the category. My fault really for not making that clear in tickets / with comments in the code.

Reply:
PS. good catch!